.github/workflows/release.yaml

```diff
@@ -0,0 +1,104 @@
+name: Release on merge to main
+on:
+  push:
+    branches:
+      - main
+
+env:
+  BOT_NAME: newrelic-coreint-bot
+  BOT_EMAIL: coreint-dev@newrelic.com
+
+permissions:
+  contents: write
+
+jobs:
+  test-release-needed:
+    name: Test if release is needed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      # Validation and generation.
+      - name: Validate that the markdown is correct
+        uses: ./validate-markdown
+      - name: Generate YAML
+        uses: ./generate-yaml
+        with:
+          excluded-dirs: .github
+
+      # Check that a release is needed.
+      - name: Check if the release is empty
+        id: empty
+        uses: ./is-empty
+      - name: Check if the release is held
+        id: held
+        uses: ./is-held
+      - name: Output if the release should be skipped
+        id: output
+        shell: bash
+        run: |
+          echo "skip=${{ steps.empty.outputs.is-empty == 'true' || steps.held.outputs.is-held == 'true' }}" >> $GITHUB_OUTPUT
+    outputs:
+      make-release: ${{ steps.output.outputs.skip }}
```

Comment:
Shouldn't it be the opposite? I'm not sure about the syntax but, something like:

```yaml
    outputs:
      make-release: ${{ steps.output.outputs.skip == 'false' }}
```

Reply:
:face_palm: Fixed: [#1de5dba](https://github.com/newrelic/release-toolkit/pull/187/commits/1de5dba1c70a18b29d48aad48560126a34418bd3)